Xamarin.Essentials/Screenshot/Screenshot.android.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Android;
+using Android.App;
+using Android.Content;
+using Android.Content.Res;
+using Android.Graphics;
+using Android.OS;
+using Android.Telephony;
+using Android.Util;
+using Android.Views;
+using Java.Util;
+using Environment = Android.OS.Environment;
+using Path = System.IO.Path;
+using Uri = Android.Net.Uri;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Screenshot
+    {
+        static bool PlatformCanCapture => !Platform.WindowManager.DefaultDisplay.Flags.HasFlag(DisplayFlags.Secure);
+
+        static string GetTempFileName() => Path.Combine(Environment.ExternalStorageDirectory.Path, Path.GetTempFileName());
```

Comment:
I am thinking that this method might be better served on the `FileSystem` type.

Reply:
Definitely, I used similar code in 2 other active branches. Does adding the method to the platform specific Filesystem class with internal visibility sound okay?

Reply:
I have a comment on `ScreenshotFile` but looking at this, if we do go the file route, we should probably use the OS cache location (`FileSystem.CacheDirectory` or something) rather than the external storage - the user may not have permissions or something. We don't have to worry about permissions for sharing as the `Share` API does all that.